covid19_sfbayarea/utils.py

```diff
@@ -46,3 +48,23 @@ def parse_datetime(date_string: str, timezone: Optional[tzinfo] = PACIFIC_TIME)
         raise ValueError(f'Unknown date format: "{date_string}"')
 
     return date
+
+
+def assert_equal_sets(a: Iterable, b: Iterable, description: str = 'items') -> None:
+    """
+    Raise a nicely formatted exception if the two arguments do not contain the
+    same items. Arguments can be any iterable; the order of items in them does
+    not matter (they are treated like sets).
+    """
+    a_set = isinstance(a, set) and a or set(a)
+    b_set = isinstance(b, set) and b or set(b)
```

Comment:
```suggestion
    a_set = set(a)
    b_set = set(b)
```
Maybe I'm looking at this the wrong way, but it seems to me when you pass a `set` into `set` it doesn't change it, so the check is unnecessary (it would fail with an error if it's not an `iterable` anyway):
``` python
a = set([0])
assert set(a) == a
```

Reply:
I was trying to avoid the overhead of constructing a new set:

```py
a = set([0])
assert set(a) == a  # True, but...
assert set(a) is a  # False
```